application/controllers/bid_details.py

```diff
@@ -68,11 +69,12 @@
 )
 bid_detail_post_parser.add_argument(
     'created_at',
-    type=datetime,
+    type=int,
     required=False,
```

Comment:
Interesting. But this makes perfect sense & should be something the FE can work with.

Reply:
Agreed. JS has a library that will interpret the timestamp much simpler than if we try to do any string or datetime work on the BE.